ksql-execution/src/main/java/io/confluent/ksql/execution/codegen/CodeGenUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.execution.codegen;
+
+import io.confluent.ksql.schema.ksql.Column;
+import io.confluent.ksql.util.KsqlPreconditions;
+
+final class CodeGenUtils {
+
+  private static final String PARAM_PREFIX = "var";
+
+  private CodeGenUtils() {
+  }
+
+  static String paramName(final Column column) {
+    KsqlPreconditions.checkArgument(
```

Comment:
This throws an `KsqlException` which is reserved for user errors.  This is not a user error...